Source/EasyNetQ/Consumer/HandlerRunner.cs

```diff
@@ -141,8 +141,17 @@ private string BuildErrorMessage(ConsumerExecutionContext context, Exception exc
 
         public void Dispose()
         {
-            consumerErrorStrategy.Dispose();
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
GC.SuppressFinalize() won't do anything because this class doesn't have a finalizer. I think all you want to do here is to add a check for null for consumerErrorStrategy:
`public void Dispose()
{
    consumerErrorStrategy?.Dispose();
}
`